schema.sql

```diff
@@ -1,5 +1,84 @@
 /* Database schema to keep the structure of entire database. */
-
+-- \c vet_clinic
+DROP TABLE IF EXISTS animals;
 CREATE TABLE animals (
-    name varchar(100)
+    id INT PRIMARY KEY NOT NULL,
+    name varchar(100) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight DECIMAL
 );
+
+INSERT INTO animals (id, name,date_of_birth,escape_attempts,neutered,weight) VALUES
+(1, 'Agumon','2020-02-03',0,false,'10.23'),
+(2, 'Gabumon','2018-11-15',2,true,'8'),
+(3, 'Pikachu', '2021-01-07',1,false, '15.04'),
+(4, 'Devimon', '2017-05-12', 5, true, '11');
```

Comment:
- Kindly move the data insertion code on lines 13-17 to the `data.sql` file. Below 👇 is a screenshot of the project requirement.

![](https://i.imgur.com/aZifjIU.png)

- On line 14, kindly note that `Agumon` was neutered. Therefore, please change the boolean value of Agumon's neutered column to `true`.

-  You can also add spaces after each comma `,` to make the code more readable. 😉

Reply:
< Actioned />